PowerEditor/src/Notepad_plus.cpp

```diff
@@ -3957,10 +3958,10 @@ bool Notepad_plus::doBlockComment(comment_mode currCommentMode)
 		size_t lineEnd = _pEditView->execute(SCI_GETLINEENDPOSITION, i);
 
 		// empty lines are not commented
-		if (lineIndent == lineEnd)
+		if (lineIndent == lineEnd && !nppGUI._lineCoomentIncBlankLines)

```

Comment:
You misspelled `_lineCommentIncBlankLines`.
But `_lineCommentBlankLines` is more consistent with the description of the parameter and `Inc` might be associated with *increment* instead of *include*.